privacyidea/models.py

```diff
@@ -3152,6 +3152,38 @@ def __init__(self, timestamp, key, value):
         #self.save()
 
 
+class Serviceid(TimestampMethodsMixin, db.Model):
+    """
+    The serviceid table contains the defined service IDs. These service ID
+    describe services like "webservers" or "dbservers" which e.g. request SSH keys
+    from the privacyIDEA system.
+    """
+    __tablename__ = 'serviceid'
+    __table_args__ = {'mysql_row_format': 'DYNAMIC'}
+    id = db.Column(db.Integer, Sequence("serviceid_seq"), primary_key=True,
+                   nullable=False)
+    name = db.Column(db.Unicode(255), default='',
+                     unique=True, nullable=False)
+    Description = db.Column(db.Unicode(2000), default='')
+
+    @log_with(log)
+    def __init__(self, servicename, description=None):
+        self.name = servicename
+        self.Description = description
+
+    def save(self):
+        si_func = Serviceid.query.filter_by(name=self.name).first
+        si = si_func()
```

Comment:
Because this is the way we did it with all other `*_func()?` methods in models.py. In most of the cases it was necessary, since it was used more than once.

Reply:
Done in 35835c5e